ui/app/templates/components/select-dropdown.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="field">
+  <label
+    for="select-dropdown-{{this.elementId}}"
+    class="is-label"
+    data-test-select-label
+    >
+    {{dropdownLabel}}
+  </label>
+  <div class="control {{if isInline "select is-inline-block"}}">
+    <div class="select {{if isFullwidth "is-fullwidth"}}">
+    <select
+      class="select"
+      id="select-dropdown-{{this.elementId}}"
+      name="selectedItem"
+      onchange={{action "onSelectItem"}}
```

Comment:
```suggestion
      onchange={{action this.onChange value="target.value"}}
```
If we call the `onChange` like that, then you can remove the actions block in the component. There's a bit about the `value` arg in the docs here: https://api.emberjs.com/ember/3.10/classes/Ember.Templates.helpers/methods/action?anchor=action